src/main/java/com/epam/test_generator/controllers/StepSuggestionController.java

```diff
@@ -2,14 +2,22 @@
 
 import com.epam.test_generator.dto.StepSuggestionDTO;
 import com.epam.test_generator.services.StepSuggestionService;
-import io.swagger.annotations.*;
+import io.swagger.annotations.ApiImplicitParam;
```

Comment:
It is good idea to format code according to codestyle, but it makes code harder to review, and furthermore, it will become much bigger pain to merge this code at some point. Please, revert all code-formatting changes. Technical debt tasks are present in all sprints, move code-formatting changes to separate task and submit it as separate pull request. Further reading http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/

Reply:
Done